app/view/home/index.js

```diff
@@ -1 +1,45 @@
 'use strict';
+require('./_home.scss');
+
+const angular = require('angular');
+const appShooter = angular.module('appShooter');
+
+appShooter.controller('HomeController', ['$log', 'scorecardService', HomeController]);
+
+function HomeController($log, scorecardService){
+  console.log('homeCtrl hit');
+
+  this.buttonTxt = 'View All Scorecards';
+  this.displayRecentCards = true;
+
+  this.toggle = function(){
+    this.displayRecentCards = !this.displayRecentCards;
+    if (this.displayRecentCards === true) {
+      this.buttonTxt = 'View All Scorecards';
+      return this.displayScorecards = this.competitions.slice(0, 5);
```

Comment:
This is a great use case for using angular filters,
we didnt cover them very much but yall should check them out and think about using one here :)